packages/editor/src/components/parts/common/customfield/StartCustomFieldTable.tsx

```diff
@@ -17,20 +28,30 @@ type StartCustomFieldTableProps = {
 const EMPTY_STARTCUSTOMSTARTFIELD: StartCustomStartField = { name: '', value: '' } as const;
 
 const StartCustomFieldTable = ({ data, onChange }: StartCustomFieldTableProps) => {
+  const { context } = useEditorContext();
+  const path = usePath();
+  const predefinedCustomField: ComboboxItem[] = useMeta(
+    'meta/workflow/customFields',
+    { context, type: toWorkflowType(path) },
```

Comment:
it is always type: 'START' here right?

Reply:
Yes, you're right, the StartCustomFieldTable is effectively only used in the RequestPart. Should I just always set START as the type in this case?

Reply:
yes